src/libraries/Microsoft.PowerFx.Core/Binding/INameResolver.cs

```diff
@@ -66,6 +66,15 @@ internal interface INameResolver
         // Looks up the parent control for the current context.
         bool LookupParent(out NameLookupInfo lookupInfo);
 
+        /// <summary>
+        /// In Canvas specifically, this is used to retrieve the full, derived, type
```

Comment:
>Canvas [](http://example.com/codeflow?start=15&length=6)

Can you say Power Apps here instead? We may get other products in future which will also have some kind of Canvas.